src/helpers/directories.rs

```diff
@@ -9,21 +9,23 @@ pub fn get_home_dir() -> Result<PathBuf> {
         return Ok(PathBuf::from(home_str));
     }
 
-    let mut home_str = if cfg!(target_os = "macos") {
-        "/Users/".to_string()
+    let mut home_str = PathBuf::new();
+
+    if cfg!(target_os = "macos") {
+        home_str.push("/Users/");
     } else {
-        "/home/".to_string()
+        home_str.push("/home/")
     };
-    if let Ok(value) = std::env::var("SUDO_USER") {
-        home_str.push_str(&value);
 
+    if let Ok(value) = std::env::var("SUDO_USER") {
+        home_str.push(&value);
         return Ok(PathBuf::from(home_str));
```

Comment:
Change this to `return Ok(home_str);`

Reply:
done, also changed it for the windows cfg